src/Book.js

```diff
@@ -0,0 +1,54 @@
+import React from "react"
+
+const Book = (props) => {
+    return (
+      <div className="bookshelf-books">
+        <ol className="books-grid">
+          {props.filteredBooks.length > 0 &&
+          props.filteredBooks.map(book => (book.shelf !== "do_not_show" &&
```

Comment:
`Book` implies a singular object and it looks like this component is displaying more than one object (book). Maybe `BookShelf` would be a better name? Lines 10-47 look more like a `Book` to me. Also some of our teammates decided to put the options (lines 20-38) in its own component, although there is some debate about that being necessary.